### PR TITLE
Add the log message length rule

### PR DESCRIPTION
**Pull request by idesi**

### CODE REVIEW

**Comment by AzinInALoop** [comment]
Is there a card for some context about PR to be attached ?

File: package-lock.json [hunk 1]
```diff
@@ -1,13 +1,13 @@
 {
   "name": "@roadmunk/eslint-config-roadmunk",
-  "version": "3.5.0",
```
Comment (AzinInALoop) [inline package-lock.json]: I am not quite sure if committing package-lock.json is necessary ?

Comment (idesi) [reply package-lock.json]: AFAIK it is a recommended practice to commit package lock file. This allows everyone on the team to get the exact same dependency tree. If `npm install` changes your node_modules, a committed lock file also makes it easy to spot that.

**Comment by idesi** [comment]
This is simply updating the config to use the latest version of our ESLint plugin so there's no card for it. 

If you'd like to know more about the change that was made to the plugin then here is the card https://roadmunk.atlassian.net/secure/RapidBoard.jspa?projectKey=DB&rapidView=118&selectedIssue=DB-10 @AzinInALoop

**Pull request merged**